Where_Is_My_Wife/Assets/_Main/Scripts/UI/MultilingualText.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using TMPro;
+using UnityEngine;
+using WhereIsMyWife.Managers;
+
+public class MultilingualText : MonoBehaviour
```

Comment:
Not needed

Reply:
96665c08c08aa3776ec96d7030a7f19c1066d17c